dojo/settings/settings.dist.py

```diff
@@ -756,12 +756,25 @@ def get_secret(secret_name):
 # Whether to use a secure cookie for the session cookie. If this is set to True,
 # the cookie will be marked as secure, which means browsers may ensure that the
 # cookie is only sent with an HTTPS connection.
-SESSION_COOKIE_SECURE = env("DD_SESSION_COOKIE_SECURE")
-SESSION_COOKIE_SAMESITE = env("DD_SESSION_COOKIE_SAMESITE")
+DD_SESSION_COOKIE_SECURE = env("DD_SESSION_COOKIE_SECURE")
+DD_SESSION_COOKIE_SAMESITE = env("DD_SESSION_COOKIE_SAMESITE")
```

Comment:
Why do you change the name of variable ?

Reply:
sorry, it's my mistake